ostree-raw-image.sh

```diff
@@ -0,0 +1,713 @@
+#!/bin/bash
+set -euox pipefail
+
+# work with mock osbuild-composer repo
```

Comment:
If you need this helper, please update repo url to rhel 8.5. Otherwise, remove them please.

Reply:
No, I don't need it, removed, thanks!